app/Http/Controllers/StudentsCouncil/EpistolaController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers\StudentsCouncil;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Mail;
+use App\Mail\EpistolaCollegii;
+use Image;
+
+use App\Models\EpistolaNews;
+
+class EpistolaController extends Controller
+{
+    public function index()
```

Comment:
Should this be publicly visible?

Reply:
No, I will fix the tests.

Reply:
Just to clarify, right now there are no authorization checks, so it's visible to all verified user. My question is, is this intended? If so, the test should be modified. If not all users should see this, an authorization check should be added.